sources/mysql/data.go

```diff
@@ -29,10 +29,6 @@ import (
 	"github.com/cloudspannerecosystem/harbourbridge/spanner/ddl"
 )
 
-// TODO: mysql/data.go and postgres/data.go share significant chunks of common code.
-// The key differences are in convArray and convTimestamp. Consider refactoring using
-// a go interface that provides the basic conv functions for each type.
-
```

Comment:
While the methods do similar things, the parameters and implementations of the methods are quite different. It might actually lead to more complex code merging them

Reply:
While these methods do similar things, the parameters and implementations are quite different. Merging these into a utility method might actually make the code a little more complicated.